Tessy/Chapter_1/exercise_1.12.6

```diff
@@ -0,0 +1,9 @@
+Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
+[GCC 5.4.0 20160609] on linux
+Type "copyright", "credits" or "license()" for more information.
```

Comment:
You forgot this part: `Type 6 + 4 * 9 at the Python prompt and hit enter. Record what happens.`. I was expecting to see you type the expression on the terminal first before moving on with executing the script.